drivers/network/dd/b57xx/CMakeLists.txt

```
@@ -0,0 +1,19 @@
+
+add_definitions(
+    -DNDIS50_MINIPORT
+    -DNDIS_MINIPORT_DRIVER
+    -DNDIS_LEGACY_MINIPORT)
```

```suggestion
    -DNDIS_MINIPORT_DRIVER
    -DNDIS51_MINIPORT=1)
```
I would suggest to target NDIS 5.1